lib/views/create-bug.js

```diff
@@ -221,8 +223,33 @@ function submitBug() {
   });
 }
 
+function createAttachmentRow(name) {
+  var DEFAULT_CHECKED = [
+    'dev-log-main.log', 'properties.log', 'system-b2g-application.ini',
+    'logshake-screenshot.png', 'screenshot.png'
+  ];
+  var label = document.createElement('label');
+  var checkbox = document.createElement('input');
+  var span = document.createElement('span');
```

Comment:
The initialization of these elements is interlaced. It would be a lot clearer if they were separated into blocks for each element.